lib/Alien/Base/ModuleBuild.pm

```diff
@@ -3,7 +3,7 @@ package Alien::Base::ModuleBuild;
 use strict;
 use warnings;
 
-our $VERSION = '1.07';
+our $VERSION = '1.08';
```

Comment:
Please don't update the version number, I do that as part of the release process.

Reply:
Sure.